recipe/meta.yaml

```diff
@@ -26,7 +24,7 @@ requirements:
   run:
```

Comment:
jinja2 is not required in host
<61 is not required for setuptools (and this will prevent 3.12 builds)